### PR TITLE
Do not write tels with trigger in pointing table

### PR DESCRIPTION
**Pull request by maxnoe**
This column is not needed (even unwanted there) and basically makes merging impossible.

### CODE REVIEW

**Comment by kosack** [comment]
why was it in there to begin with? Just to write the mean trigger time?   Perhaps we need a better place to keep track of subarray event time.  It's not even clear we have a good definition of subarray time : i.e. is it the the mean telescope time?

**Review by kosack — DISMISSED** [review]

**Comment by maxnoe** [comment]
Yes, to write the time. Which is a hack for simtel array files since these do not have monitoring data with timestamps, so we use the event time (which makes sense in trigger).

**Comment by maxnoe** [comment]
This probably needs further fixing as soon as I can use the LST event source with the dl1 tool and inspect the output.

**Review by kosack — APPROVED** [review]

File: ctapipe/tools/stage1.py [hunk 1]
```diff
@@ -483,7 +483,7 @@ def _process_events(self, writer):
     def _write_telescope_event(self, writer, event):
         """
         add entries to the event/telescope tables for each telescope in a single
-        event
+        even
```
Comment (vuillaut) [inline ctapipe/tools/stage1.py:486]: event was right, no?

File: ctapipe/tools/stage1.py [hunk 2]
```diff
@@ -623,6 +624,10 @@ def _setup_writer(self, writer: HDF5TableWriter):
                 writer.exclude(
                     f"/dl1/event/telescope/images/{table_name}", "image_mask"
                 )
+
+            writer.exclude(
+                f"/dl1/monitoring/telescope/trigger/{table_name}", "trigger_pixels"
+            )
```
Comment (vuillaut) [inline ctapipe/tools/stage1.py:630]: isn't this important for real data?

Comment (maxnoe) [reply ctapipe/tools/stage1.py:630]: At the DL1 level you want to know which trigger patches contributed to the trigger decision?

Comment (maxnoe) [reply ctapipe/tools/stage1.py:630]: Then we would need to convert it to a mask. Currently the list is not writable and this exclude is only to silence the warnings.

**Review by vuillaut — APPROVED** [review]

**Pull request merged**